pom.xml

```diff
@@ -793,6 +793,7 @@
                         </executions>
                         <configuration>
                             <doclint>none</doclint>
+                            <detectOfflineLinks>false</detectOfflineLinks>
```

Comment:
It's in parent pom, this will effect all the sub-modules.
Currently only  the common module and the dist module have problem.
Can we just update these 2 modules?